### ISSUE TITLE
Recommend using `npx` over installing react-native-cli globally

### ISSUE THREAD

**Issue by tido64** [microsoft/react-native-windows-samples#25]
According to React Native CLI [documentation](https://github.com/react-native-community/cli/blob/master/docs/init.md), you're not supposed to install `react-native-cli` globally any longer. The recommended approach is to use `npx`, e.g.:

```bash
npx react-native init MyApp
# or
npx react-native@^0.60 init MyApp
```

**Comment by chrisglein** [comment]
@kikisaints Can you try the getting started steps using npx and confirm whether we're ready to change the recommended steps or not yet? And if we are, updated the documentation.

**Comment by kikisaints** [comment]
This has been addressed as of #33 and #31 .
